### PR TITLE
cpdp-1400: check all answers, submit and view multiple disposals returns

### PR DESCRIPTION
**Pull request by AndyHWChung**

### CODE REVIEW

File: app/uk/gov/hmrc/cgtpropertydisposalsfrontend/models/returns/CompleteReturn.scala [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+import uk.gov.hmrc.cgtpropertydisposalsfrontend.models.returns.ExamplePropertyDetailsAnswers.CompleteExamplePropertyDetailsAnswers
+import uk.gov.hmrc.cgtpropertydisposalsfrontend.models.returns.ExemptionAndLossesAnswers.CompleteExemptionAndLossesAnswers
+import uk.gov.hmrc.cgtpropertydisposalsfrontend.models.returns.MultipleDisposalsTriageAnswers.CompleteMultipleDisposalsTriageAnswers
+import uk.gov.hmrc.cgtpropertydisposalsfrontend.models.returns.ReliefDetailsAnswers.CompleteReliefDetailsAnswers
+import uk.gov.hmrc.cgtpropertydisposalsfrontend.models.returns.SingleDisposalTriageAnswers.CompleteSingleDisposalTriageAnswers
+import uk.gov.hmrc.cgtpropertydisposalsfrontend.models.returns.UploadSupportingDocuments.CompleteUploadSupportingDocuments
+import uk.gov.hmrc.cgtpropertydisposalsfrontend.models.returns.YearToDateLiabilityAnswers.CalculatedYTDAnswers.CompleteCalculatedYTDAnswers
+import uk.gov.hmrc.cgtpropertydisposalsfrontend.models.returns.YearToDateLiabilityAnswers.NonCalculatedYTDAnswers.CompleteNonCalculatedYTDAnswers
+
+sealed trait CompleteReturn
```
Comment (mikail-khan) [inline app/uk/gov/hmrc/cgtpropertydisposalsfrontend/models/returns/CompleteReturn.scala]: We now have:

1. SingleDisposalDraftReturn

1. MultipleDisposalsDraftReturn

1. CompleteMultipleDisposalsReturn

1. CompleteSingleDisposalReturn

Should rename MultipleDisposalsDraftReturn and SingleDisposalsDraftReturn to DraftSingleDisposalReturn and DraftMultipleDisposalsDraftReturn to be consistent with the naming

Comment (AndyHWChung) [reply app/uk/gov/hmrc/cgtpropertydisposalsfrontend/models/returns/CompleteReturn.scala]: yeah, we can do that - that makes sense

Comment (AndyHWChung) [reply app/uk/gov/hmrc/cgtpropertydisposalsfrontend/models/returns/CompleteReturn.scala]: done and pushed up

**Comment by AndyHWChung** [comment]
for some reason there's merge conflicts now - I'll fix them

File: app/uk/gov/hmrc/cgtpropertydisposalsfrontend/models/returns/CompleteReturn.scala (hunk 1 above)
Comment (mikail-khan) [reply app/uk/gov/hmrc/cgtpropertydisposalsfrontend/models/returns/CompleteReturn.scala]: does the stub service need to change?

Comment (AndyHWChung) [reply app/uk/gov/hmrc/cgtpropertydisposalsfrontend/models/returns/CompleteReturn.scala]: not until we want to be able to view multiple returns - that will come next when I do the BE updaes to read the return back from DES/ETMP

**Comment by mikail-khan** [comment]
Yes, because now you have altered files which have changed in master and now are conflicting with yours locally

**Comment by AndyHWChung** [comment]
conflicts fixed - I ran the tests a few times and they're passing

**Pull request merged**